packages/docs/src/examples/Multichain.tsx

```diff
@@ -22,14 +20,14 @@ ReactDOM.render(
 )
 
 export function App() {
-  const mainnetBalance = useEtherBalance(address, { chainId: Mainnet.chainId })
   const arbitrumBalance = useEtherBalance(address, { chainId: Arbitrum.chainId })
+  const zkSyncBalance = useEtherBalance(address, { chainId: ZkSyncTestnet.chainId })
 
   return (
     <>
       <div> Account address: {address}</div>
-      <div> Balance on Mainnet: {mainnetBalance && formatEther(mainnetBalance)} Eth </div>
```

Comment:
Maybe leave Mainnet and display balance on three chains?

Reply:
Marek wanted zksync and arbitrum only, but i think we can add third balance.